src/components/input/Input.tsx

```diff
@@ -15,24 +16,34 @@ interface IInput {
   value?: string;
 }
 
-const Input: FC<IInput> = ({ accept, className, name, value, type = 'text', title, errorText, placeholder, onChange }): ReactElement => {
-
-  return (
-    <div className={cn('one-input-block', className, { error: errorText })}>
-      {title && <label htmlFor={name} className='title'>{title}</label>}
-      <input
-        accept={accept}
-        id={name}
-        className='input'
-        onChange={onChange}
-        disabled={!onChange}
-        placeholder={placeholder}
-        name={name}
-        value={value}
-        type={type} />
-      <span className='error-message'>{errorText}</span>
-    </div>
-  );
-};
+const Input: FC<IInput> = ({
+  accept,
+  className,
+  name,
+  value,
+  type = 'text',
+  title,
+  errorText,
+  placeholder,
+  onChange,
+  onBlur
+}): ReactElement => (
+  <div className={cn('one-input-block', className, { error: errorText })}>
+    {title && <label htmlFor={name} className='title'>{title}</label>}
+    <input
+      accept={accept}
+      id={name}
+      className='input'
+      onChange={onChange}
+      disabled={!onChange}
+      placeholder={placeholder}
+      name={name}
+      value={value}
+      type={type}
+      onBlur={onBlur}
+    />
```

Comment:
👍🏻
Классно отрефачила! Но давай ещё все коллбеки в конце сгруппируем
```suggestion
      disabled={!onChange}
      placeholder={placeholder}
      name={name}
      value={value}
      type={type}
      onChange={onChange}
      onBlur={onBlur}
    />
```

Reply:
Исправила